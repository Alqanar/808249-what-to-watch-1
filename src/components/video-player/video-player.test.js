import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player.tsx";


const film = {
    posterLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_843650.jpg`,
    trailer: `https://youtu.be/TcMBFSGVi1c`
};

it(`Video player is correctly renders`, () => {
    const tree = renderer
        .create(<VideoPlayer
            film={film}
        />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
