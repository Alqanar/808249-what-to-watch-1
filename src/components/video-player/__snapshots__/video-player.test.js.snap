// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player is correctly renders 1`] = `
<video
  height="175"
  loop={true}
  muted={true}
  poster="https://st.kp.yandex.net/images/film_iphone/iphone360_843650.jpg"
  src="https://youtu.be/TcMBFSGVi1c"
  width="280"
/>
`;
